packages/state/clients/cw-proposal-single.ts

```diff
@@ -422,7 +424,7 @@ export class CwProposalSingleQueryClient
     voter: string
   }): Promise<VoteResponse> => {
     return this.client.queryContractSmart(this.contractAddress, {
-      vote: {
+      get_vote: {
```

Comment:
Memorializing discord conversation to make this conditional based on contract version!